src/components/CourseSearchResults.js

```diff
@@ -8,6 +8,10 @@ const CourseSearchResults = ({ isSuccessfulSearch, searchResults }) => {
         return <SearchFailedMessage />;
     }
 
+    if (!searchResults?.length) {
```

Comment:
Very minor point, but could this be simplified?
```suggestion
    if (!searchResults) {
```

Reply:
Sounded good to me so I committed the suggestion, but it caused the tests to fail!

`!searchResults` just checks for the existence of the array I guess, and not the content. I've decided to make it `!searchResults || !searchResults.length` to be a bit more robust

Reply:
Oops... Sorry! :-/

Reply:
No problem at all, it's a learning experience!